firmware/stackchan/default-mods/mod.ts

```
@@ -3,8 +3,8 @@ import { onLaunch } from 'default-mods/on-launch'
 import type { Robot } from 'robot'
 
 export interface StackchanMod {
-  onLaunch?: () => Promise<void | boolean> | void | boolean
-  onRobotCreated?: (robot: Robot, option?: unknown) => Promise<void> | void
+  onLaunch?: () => Promise<undefined | boolean> | undefined | boolean
+  onRobotCreated?: (robot: Robot, option?: unknown) => Promise<undefined> | void
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)